### PR TITLE
Destination implementation

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've implemented the Destination and added tests.

### Quick checks:

- [x] There is no other [pull request](https://github.com/conduitio/conduit-connector-connectorName/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: destination/writer/writer.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+		table:      config.Table,
+		keyColumns: config.KeyColumns,
+	}
+
+	writer.db, err = sqlx.Open(driverName, config.DSN)
+	if err != nil {
+		return nil, fmt.Errorf("open db connection: %w", err)
+	}
+
+	u, err := url.Parse(config.DSN)
```
Comment (alyakimenko) [inline destination/writer/writer.go:59]: Thoughts out loud: wouldn't it make sense to make the `config.DSN` field `*url.URL`?

Comment (voscob) [reply destination/writer/writer.go:59]: Do you mean parse `config.DSN` and call `String()` method to get `config.DSN`?

Comment (alyakimenko) [reply destination/writer/writer.go:59]: https://github.com/conduitio-labs/conduit-connector-redshift/blob/4563a30da5181f2fd18b3a7b7e6205229749444c/config/configuration.go#L34
Yeah, I thought about changing the `Configuration.DSN`'s type to `*url.URL` and parsing it right in the `parseCommon` function. This way we can avoid parsing this field in Source and Destination.

Comment (voscob) [reply destination/writer/writer.go:59]: Hm. It could be a good idea 👍 

I think I will do it, but definitely in a different pull request. Thank you.

**Review by alyakimenko — APPROVED** [review]
Looks good to me, good job 🎉

**Pull request merged**